server/router.go

```diff
@@ -14,4 +15,9 @@ func (a *App) RegisterRoutes() {
 	r.Use(middleware.JSONContentType)
 
 	r.Get("/hello", a.HelloHandler)
+
+	r.Route("/{account}", func(r chi.Router) {
+		r.Post("/", a.AccountHandler)
+		r.Get("/", a.AccountHandler)
+	})
```

Comment:
It does seem a bit redundant to create a subrouter and then switch on the method to call out to different implementation handlers. We can revisit this later if it becomes a real problem, but that seems like extra boilerplate for no gain as compared to just linking to the implementation routers for each type of request, especially if they do end up being quite different from each other.